07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,56 @@
+.image  {
+    position: relative;
+    width: 850px;
+ }
+
+.text {
+    position: absolute;
+    bottom: 0;
+    font-size: 80px;
+    font-family: Arial;
+    color: white;
+    padding: 110px 10px;
+    width: 100%;
+    text-align: center;
+ }
+
+.container  {
+    position: relative;
+    width: 850px;
+    height: : 324px;
+    color: gray;
+ }
+
+.box1  {
+
+    position: relative;
+    float: left;
+    width: 270px;
+    height: 324px;
+ }
+
+.box1 > h1 {
+      color: black;
+}
+
```

Comment:
keep in mind that you need the space inbetween the boxes that is exactly to the specifications.